utils/evaluator.py

```diff
@@ -30,16 +28,14 @@ def __call__(self, data_loader):
         data_iter = iter(data_loader)
         total_iou = 0
         total_f_score = 0
-        self.init_progress_bar()
-        for index, (img, ground_truth) in enumerate(data_iter):
-            img_tensor = self.img_to_tensor(img)
-            prediction = self.model.forward(img_tensor)
+        iterable = tqdm(enumerate(data_iter)) if self.is_progress_bar else enumerate(data_iter)
+        for index, (img, ground_truth) in iterable:
+            prediction = self.model(img)
             self.current_intersection = torch.sum(prediction * ground_truth)
             self.total_pixels = torch.sum(ground_truth) + torch.sum(prediction)
             self.current_union = self.total_pixels - self.current_intersection
             total_iou += self.iou()
             total_f_score += self.f_score()
-            self.print_progress(index, len(data_loader))
```

Comment:
you can replace the print function with `texttable`, will do some heavy lifting too

Reply:
I removed that method. it was used to display the progress bar, sure